doc/gb-model/methodology.rst

```diff
@@ -0,0 +1,315 @@
+2. Calculate the interconnector fee (:math:`Fee`) as the market spread between GB and Europe (difference in wholesale electricity price)
+3. In each hour of the unconstrained run, calculate the the marginal plant in Europe to assign a bid/offer multiplier to that hour.
+   The marginal price combined with the multiplier gives the interconnector :math:`Bid` / :math:`Offer`.
+   It is assumed that the import/export quantity in re-dispatch is not sufficient to change the marginal plant in Europe.
+4. Apply a cost to different changes in state:
+  1. Interconnector importing & reduce imports / float / start exporting: :math:`P_{GB} - P_{foreign} \times (1 + Loss) - Bid \times (1 + Loss)`
+  2. Interconnector importing & increase imports: :math:`Fee + Offer \times (1 + Loss)`
+  3. Interconnector at float & start exporting: :math:`Fee - Bid \times (1 - Loss)`
+  4. Interconnector at float & start importing: :math:`Fee + Offer \times (1 + Loss)`
+  5. Interconnector exporting & increase exports: :math:`Fee - Bid \times (1 - Loss)`
+  6. Interconnector exporting & reduce exports / float / start importing: :math:`P_{foreign} \times (1 - Loss) - P_{GB} + Offer \times (1 - Loss)`
+
+.. note::
+
+  It is unclear from the documentation what :math:`P` refers to in the above math.
+
+
+The case of importing/exporting in the unconstrained run and then exporting/importing in the re-dispatch run is poorly represented by these costs (as it switches between two cost formulations).
+
+Plant availability
+==================
+
+Data is usually available per month.
+Some is supposedly available from the FES, although this is augmented with proprietary datasets.
+Availability reflects planned and unplanned outages and is applied to generators as a de-rating of their peak capacity.
+This is applied per month, possibly even at a higher resolution if appropriate data is available.
+It is unclear how granular plant availability has been applied, beyond monthly de-rating.
+
+Interconnectors have a constant 95% availability applied to them.
+
+Plant representation
+====================
+
+Thermal plants
+--------------
+
+Thermal plants are grouped if they have similar variable costs.
+Characteristics that are represent include "technology type, fuel type, efficiency, start-up cost,
+part load efficiency, operating cost, and availability" as well as must-run restrictions (if there are contractual obligations).
+Fuel costs do not fluctuate within the year but do vary by European country.
+The FES provides fuel and carbon price forecasts.
+
+Combined Heat and Power
+-----------------------
+
+The model can distinguish between extraction CHP and backpressure CHP.
+For each CHP plant, heat load profiles are specified.
+
+Hydropower
+----------
+
+Hydro is split in the model into Reservoir and Run-Of-River.
+Inflows are modelled as a cascade with multiple levels, with inflow expectation, the ability of generators to forecast inflows ahead of time, and actual inflow levels.
```

Comment:
does that mean we will use two different runs one with forecasted time series and the other one with actual?

Reply:
Moved this to #45